symphony/bdk/core/config/model/bdk_config.py

```diff
@@ -0,0 +1,21 @@
+from symphony.bdk.core.config.model.bdk_bot_config import BdkBotConfig
+from symphony.bdk.core.config.model.bdk_server_config import BdkServerConfig
+from symphony.bdk.core.config.model.bdk_client_config import BdkClientConfig
+from symphony.bdk.core.config.model.bdk_ssl_config import BdkSslConfig
+
+
+class BdkConfig(BdkServerConfig):
+    """Class containing the Bdk configuration
+
+    :param **config: dict Configuration parameters
+    :return self: BdkConfig instance
+    """
+    def __init__(self, **config):
+        super().__init__(scheme=config.get("scheme"), host=config.get("host"), port=config.get("port"),
```

Comment:
Please do the same as BdkCertificateConfig constructor.

Reply:
ok